.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we remove those eclipse generated files like .project,.classpath,  .settings etc. ?

Reply:
ok i will add .project , .classpath and .setting in .gitignore file and commit again.